spring/boot2-autoconfigure/src/main/java/com/linecorp/armeria/spring/AbstractArmeriaBeanPostProcessor.java

```diff
@@ -70,7 +70,7 @@
 
     protected InjectionMetadata findLocalArmeriaPortMetadata(
             String beanName, Class<?> clazz, @Nullable PropertyValues pvs) {
-        final String cacheKey = Strings.isNullOrEmpty(beanName) ? beanName : clazz.getName();
+        final String cacheKey = Strings.isNullOrEmpty(beanName) ? clazz.getName() : beanName;
```

Comment:
Oops...

Reply:
Oops...

Reply:
Slight recommendation to add `!` instead of changing the order to keep the two `beanName` closer to each other

Reply:
Seems better. flipped now.